rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1057,6 +1057,12 @@ public long getLockingCap(Object[] args) {
         return lockingCap.getValue();
     }
 
+    public byte[] getActivePowpegRedeemScript(Object[] args) {
+        logger.trace("getActivePowpegRedeemScript");
```

Comment:
Please follow logging conventions as we discussed them

Reply:
I  changed it and include new logging before and after execute the method following the conventions we put on GDrive. Let me know what you think about this new version of the method.